Assignment_08.ipynb

```diff
@@ -0,0 +1,201 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f6d644d2208>"
+     
```

Comment:
For your G function, the critical points should be computed for each distance band. Here is an example of what the G-Function should look like: http://4.bp.blogspot.com/-OhBzDT4TSZg/VVdEfKo8YQI/AAAAAAAAQ9o/84HOh0_UFHc/s1600/Fig10.tiff